packages/@lwc/synthetic-shadow/src/faux-shadow/node.ts

```diff
@@ -42,76 +42,12 @@ import {
 } from './traverse';
 import { getTextContent } from '../3rdparty/polymer/text-content';
 import { getShadowRoot, isHostElement, getIE11FakeShadowRootPlaceholder } from './shadow-root';
+import { getNodeNearestOwnerKey, getNodeOwnerKey, isNodeShadowed } from '../shared/node-ownership';
 import { createStaticNodeList } from '../shared/static-node-list';
 import { isGlobalPatchingSkipped } from '../shared/utils';
 
-const OwnKey = '$$OwnKey$$';
-const OwnerKey = '$$OwnerKey$$';
-
 export const hasNativeSymbolsSupport = Symbol('x').toString() === 'Symbol(x)';
```

Comment:
This function doesn't belong here either, I would move it to the utils. Or we should at least remove the `export` since it is only used in this file.